CONTRIBUTING.md

```diff
@@ -2,45 +2,41 @@
 
 * Identify use cases that will shape the standards and APIs
 * How to add or delete objects and/or object attributes
 * How a particular attribute should be defined
 * Report bugs you encounter when using the reference implementations
 
-
-The [Issues Page](https://github.com/ga4gh/ReadTaskTeam/issues) serves as a public forum to discuss and debate topics related to the proposed standards, formats, and APIs. It also serves as the means for collaborating with the group and making contributions such as proposing changes to the formats and APIs. See the [Issue Resolution](#issue_resolution) section below for specifics on how issues are resolved by the community.
-
-A second way to contribute to the project is to directly contribute development effort. Please refer to the next section, [Contributions and Pull Request](#pull_request), for more details.
-
 <a name="pull_request"></a>
-## Contributions and Pull Requests
+## Pull Requests
 
-The way to contribute development effort and code to the project is via GitHub pull requests. GitHub provides a nice [overview on how to create a pull request](https://help.github.com/articles/creating-a-pull-request).
+The way to contribute development effort and code to the project is via GitHub pull requests. GitHub provides a nice [overview on how to create a pull request](https://help.github.com/articles/creating-a-pull-request). Contributions typically require pull requests to each of the schemas, server and compliance repositories, although pull requests to the server may merely improve the code without affecting the API, and therefore changing the schemas or compliance tests. A set of branches across the repositories each with the same name is a branch set, e.g. the master branch in each repository forms the master branch set.
 
 Some general rules to follow:
 
 * [Fork](https://help.github.com/articles/fork-a-repo) the main project into your personal GitHub space to work on.
 * Create a branch for each update that you're working on. These branches are often called "feature" or "topic" branches. Any changes that you push to your feature branch will automatically be shown in the pull request.
+* If necessary, replicate the last two step's fork-and-branch process for each of the compliance and server repositories to create a branch set - each constiuent repository branch will be identically named.
+* Coordinate pull requests across the branch set by making them as simulatenously as possible, and [cross referencing them](http://stackoverflow.com/questions/23019608/github-commit-syntax-to-link-a-pull-request-issue).
```

Comment:
I assume the individual merging a pull request would be required to merge the other related ones at the same time - and in their absence, to ask the contributor to supply/update them - should we add this in here?